### ISSUE TITLE
An unhandled exception occurred while processing the request.

### ISSUE THREAD

**Issue by diegomodolo** [OfficeDev/TeamsFx#10416]
I've created two projects using the Teams Toolkit, but both fail when I execute with the following message:

An unhandled exception occurred while processing the request.
InvalidOperationException: Cannot find the fallback endpoint specified by route values: { page: /_Host, area: }.
Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.DynamicPageEndpointMatcherPolicy.ApplyAsync(HttpContext httpContext, CandidateSet candidates)

1. Create a new Microsoft Teams App (C#)
2. Right click on the project name, go to Teams Toolkit and click on "Prepare Teams App Dependencies"
3. Execute with F5 or clicking at "Microsoft Teams (browser)"
4. Add the application to Teams.
5. See error

**Expected behavior**
It should display the Welcome component.

![image](https://github.com/OfficeDev/TeamsFx/assets/12273213/e0ae8082-3b92-4937-9afd-af52e86cfb3a)

**VS Code Extension Information (please complete the following information):**

**CLI Information (please complete the following information):**
 - OS: Windows
 - Version 11

**Additional context**

**Comment by xiaolang124** [comment]
@diegomodolo For now, you can try adding these to your .csproj file.

```
<PropertyGroup>
  <UseRazorSourceGenerator>false</UseRazorSourceGenerator>
</PropertyGroup>
```
You may try updating your Visual Studio and .NET versions to resolve this once and for all.
Related issue: https://github.com/dotnet/aspnetcore/issues/36535

**Comment by diegomodolo** [comment]
Hi @xiaolang124 , updating VS solved the issue.
Since we are talking, are there any plans of making the project compatible with .Net 7.0 and 8.0?

**Comment by xiaolang124** [comment]
@diegomodolo Yes, we will be adding support for .NET 7 and .NET 8 in the upcoming 17.9 Preview 2 next month. This support will then be included in the stable version of 17.9. You can try this feature later.
